{
	"cve": {
		"data_type": "CVE",
		"data_format": "MITRE",
		"data_version": "4.0",
		"CVE_data_meta": {
			"ID": "CVE-2021-38875",
			"ASSIGNER": "psirt@us.ibm.com"
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": []
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/208398",
					"name": "ibm-mq-cve202138875-dos (208398)",
					"refsource": "XF",
					"tags": []
				},
				{
					"url": "https://www.ibm.com/support/pages/node/6517672",
					"name": "https://www.ibm.com/support/pages/node/6517672",
					"refsource": "CONFIRM",
					"tags": []
				}
			]
		},
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "IBM MQ 8.0, 9.0 LTS, 9.1 LTS, 9.2 LTS, 9.1 CD, and 9.2 CD is vulnerable to a denial of service attack caused by an error processing messages. IBM X-Force ID: 208398."
				}
			]
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": []
	},
	"impact": {},
	"publishedDate": "2021-11-23T20:15Z",
	"lastModifiedDate": "2021-11-23T21:15Z"
}